LsifDotnet/IndexHandler.cs

```diff
@@ -20,15 +21,18 @@ namespace LsifDotnet;
 public class IndexHandler
 {
     public static async Task Process(IHost host, FileInfo? solutionFile, FileInfo output, CultureInfo culture, bool dot,
-        bool svg, bool quiet, uint parallelism, uint index)
+        bool svg, bool quiet, uint parallelism, uint index, string[] excludedFiles)
     {
         var logger = host.Services.GetRequiredService<ILogger<IndexHandler>>();
         ConfigLoggingLevel(host, quiet);
         PrintDescription(logger);
 
+
         var solutionFilePath = solutionFile?.FullName ?? FindSolutionFile();
 
         await host.Services.GetRequiredService<MSBuildWorkspace>().OpenSolutionAsync(solutionFilePath);
+        MatcherFactory.getMatcher().AddIncludePatterns(excludedFiles);
```

Comment:
You can check how  [IdentifierCollectorFactory](https://github.com/tcz717/LsifDotnet/blob/main/LsifDotnet/Roslyn/IdentifierCollectorFactory.cs) works and how to use 

- https://github.com/tcz717/LsifDotnet/blob/main/LsifDotnet/Program.cs#L86 
- https://github.com/tcz717/LsifDotnet/blob/main/LsifDotnet/Lsif/DataFlowLsifIndexer.cs#L23

Also, in C# you should use GetMatcher as the name.